src/gui/properties/speedwidget.cpp

```diff
@@ -116,53 +116,42 @@ SpeedWidget::SpeedWidget(PropertiesWidget *parent)
 
     loadSettings();
 
-    m_isUpdating = true;
-    m_updateFuture = QtConcurrent::run(this, &SpeedWidget::update);
+    m_updateTimer=new QTimer(this);
+    connect(m_updateTimer, SIGNAL(timeout()), this, SLOT(update()));
```

Comment:
@dzmat:  Could you please upgrade this to the Qt5 `connect` syntax?  

See [<https://wiki.qt.io/New_Signal_Slot_Syntax>].

Reply:
@dzmat:  Also mention in the commit message that this change is specifically for `speedwidget`.

Reply:
fixed style and commit message